OpenUtau.Core/PlaybackManager.cs

```diff
@@ -241,7 +242,8 @@ public void OnNext(UCommand cmd, bool isUndo) {
                 DocManager.Inst.ExecuteCmd(new SetPlayPosTickNotification(0));
             }
             if (cmd is PreRenderNotification || cmd is LoadProjectNotification) {
-                if (Util.Preferences.Default.PreRender) {
+                // Always prerender when it's connected to daw
```

Comment:
- Is using prerendering ok?:

The daw integration uses pre-render audios, but I don't know if there is something that prerender misses.